### DISCUSSION TITLE
Jellyfin Server with remote libraries: avoid erasing database on share link drop

### DISCUSSION THREAD

**Discussion by alex-devg** [jellyfin/jellyfin#6668] [Q&A]
Maybe my title isn't really clear.

Here is how the arch looks:

- **SERVER: JellyfinServer on RPi4**
- - /media/videos/NFS-PC-Movies (nfs://PC/export/movies)
- - /media/videos/NFS-NAS-TV-Shows (nfs://NAS/export/tv-shows)
- DATA:
- - **PC:**   /export/movies
- - **NAS:** /export/tv-shows

If for some reason one the NFS-Share link is down during the scheduled scan of the libraries it will erase all the content of the library that was on this NFS-Share. Which is the normal behavior expected as we don't want content in the library that are not available. It's less of a problem now that I've put all metadata together with the videos (NFO / imgs), so it's fast to include again the content to the library. I still loose the "recently added" which will include all the content now.  

But I was wondering if there could be a solution for this, like stop autoscan when a mount is down?

> I saw this: https://github.com/jellyfin/jellyfin/issues/1714

But I didn't find any Feature Request related to this thread, so I don't know if there's a solution to this not a bug "problem" except disabling the scheduled scan? 

I didn't try this, but could I sync all NFO files in the folder /media/videos/XXX/ and expect Jellyfin to think that the content is still here? I mean in the actual folder, not virtual mounted one, so if the mount is down I'll have the content of the local folder with the NFOs files?

**Comment by cactushydrocodone** [comment]
You could probably "fake" the filesystem with a cache overlay.
So that jellyfin "sees" the files, but cannot open / read them, as they are not there.
